perfkitbenchmarker/benchmark_sets.py

```diff
@@ -190,7 +191,7 @@ def GetBenchmarksFromFlags():
     else:
       benchmark_names.add(benchmark)
 
-  # Expand recursive sets
+# Expand recursive sets
```

Comment:
It was like that, but `flake8` gave me an error that `indentation is not a multiple of four (comment)`. So I changed it to this. What should I do here?